openshift-ci/scripts/oci-model-registry-deploy.sh

```diff
@@ -157,6 +158,7 @@ run_deployment_tests() {
 
 # Main function for orchestrating deployments
 main() {   
+    ./install_yq.sh
```

Comment:
at line 32 in `openshift-ci/scripts/install_yq.sh` you "Call the function when the script is sourced".

you source it at line 9.

didn't understand why you need this one here?

Reply:
You are correct Matteo, changing it now.

Reply:
I think it might be easier to install yq via the openshift-ci deployment.